airbyte-webapp/package.json

```diff
@@ -180,27 +180,13 @@
     "minimatch": "^3.0.5"
   },
   "lint-staged": {
-    "src/**/*.{js,jsx,ts,tsx}": [
-      "eslint --fix"
-    ],
-    "src/**/*.{css,scss,md,json}": [
-      "prettier --write"
-    ],
-    "{public,src}/**/*.{css,scss}": [
-      "stylelint --fix"
-    ]
+    "src/**/*.{js,jsx,ts,tsx}": ["eslint --fix"],
+    "src/**/*.{css,scss,md,json}": ["prettier --write"],
+    "{public,src}/**/*.{css,scss}": ["stylelint --fix"]
```

Comment:
Curious why this changed. Was linting not run here previously?

Reply:
Apparently not! This file is outside of `src/`, so `lint-staged` won't ensure linting; I guess just have a more aggressive auto-format setup in my editor than most. It'll be nice to get the linted version merged, though: I've skipped adding these lines to a bunch of commits already.

Reply:
I think there might be some more issues with the setup. Even if I run prettier over package.json manually, it will end up with the previous format not the one your editor ended up with. Can we check if there's anything in your setup that might format this with another formatter than prettier?